### ISSUE TITLE
Kusama balance query does not work in develop

### ISSUE THREAD

**Issue by LefterisJP** [rotki/rotki#2340]
## Problem Definition

**Thankfully it works with the released version we have in v1.13.2, so users are not affected**
Presumably because we actually copy the json files during packaging?

So I just tried to query my kusama test accounts with develop instead of v1.13.2 and got this error in the UI:

```
Error at querying blockchain balances: The backend query task died unexpectedly: Current offset: 72 / length: 76
```

Interesting here is that the error did not kill ALL blockchain balances, just the Kusama one. So we need to edit the title to specify Kusama up there.

The stack trace is:

```
2021-02-11T00:16:43.984Z: Traceback (most recent call last):
  File "src/gevent/greenlet.py", line 906, in gevent._gevent_cgreenlet.Greenlet.run
  File "/home/lefteris/w/rotkehlchen/rotkehlchen/api/rest.py", line 268, in _do_query_async
    result = getattr(self, command)(**kwargs)
  File "/home/lefteris/w/rotkehlchen/rotkehlchen/api/rest.py", line 563, in _query_blockchain_balances
    ignore_cache=ignore_cache,
  File "/home/lefteris/w/rotkehlchen/rotkehlchen/utils/interfaces.py", line 115, in wrapper
    result = f(wrappingobj, *args, **kwargs)
  File "/home/lefteris/w/rotkehlchen/rotkehlchen/utils/interfaces.py", line 74, in wrapper
    result = f(wrappingobj, *args, **kwargs)
  File "/home/lefteris/w/rotkehlchen/rotkehlchen/chain/manager.py", line 461, in query_balances
    self.query_kusama_balances()
  File "/home/lefteris/w/rotkehlchen/rotkehlchen/utils/interfaces.py", line 115, in wrapper
    result = f(wrappingobj, *args, **kwargs)
  File "/home/lefteris/w/rotkehlchen/rotkehlchen/utils/interfaces.py", line 74, in wrapper
    result = f(wrappingobj, *args, **kwargs)
  File "/home/lefteris/w/rotkehlchen/rotkehlchen/chain/manager.py", line 506, in query_kusama_balances
    account_amount = self.kusama.get_accounts_balance(self.accounts.ksm)
  File "/home/lefteris/w/rotkehlchen/rotkehlchen/chain/substrate/manager.py", line 569, in get_accounts_balance
    balances[account] = self.get_account_balance(account)
  File "/home/lefteris/w/rotkehlchen/rotkehlchen/chain/substrate/manager.py", line 87, in wrapper
    result = func(manager, *args_, **kwargs_)
  File "/home/lefteris/w/rotkehlchen/rotkehlchen/chain/substrate/manager.py", line 542, in get_account_balance
    node_interface=node_interface,
  File "/home/lefteris/w/rotkehlchen/rotkehlchen/chain/substrate/manager.py", line 284, in _get_account_balance
    params=[account],
  File "/home/lefteris/.virtualenvs/rotkipy37/lib/python3.7/site-packages/substrateinterface/base.py", line 1204, in query
    obj.decode()
  File "/home/lefteris/.virtualenvs/rotkipy37/lib/python3.7/site-packages/scalecodec/base.py", line 357, in decode
    raise RemainingScaleBytesNotEmptyException('Current offset: {} / length: {}'.format(self.data.offset, self.data.length))
scalecodec.exceptions.RemainingScaleBytesNotEmptyException: Current offset: 72 / length: 76
2021-02-11T00:15:31Z <Greenlet at 0x7f448374def0: <bound method RestAPI._do_query_async of <rotkehlchen.api.rest.RestAPI object at 0x7f44847d3350>>('_query_blockchain_balances', 5, blockchain=<SupportedBlockchain.KUSAMA: 'KSM'>, ignore_cache=False)> failed with RemainingScaleBytesNotEmptyException
```

## Task

Figure out why this is happening and fix it in a proper way. Proper here means that we should **never** get failures in the substrate querying library using the same code and not having upgraded its version. That's now how a library should work.

**Comment by vnavascues** [comment]
Related #2196

**Comment by vnavascues** [comment]
@LefterisJP I think I know what is going on.
The library `py-substrate-interface` does not have `==` requirement on `py-scale-codec`, but `>=`. Every time you install `py-susbtrate-interface` (e.g. building the binaries) it installs the latest version of `py-scale-codec` which contains the latest type registries. This is the reason why I've experienced the same problem (`RemainingScaleBytesNotEmptyException`) you described here with my stale dev venv (`py-substrate-interface` v0.11.8) but either re-installing it or upgrading it to the new version (v0.11.16) it did work... as `py-scale-codec` was upgraded to v0.10.57, pulling compatible registries.
```
When on-chain runtime upgrades occur, types used in call- or storage functions can be added or modified. Therefor it is important to keep the type registry presets up to date, otherwise this can lead to decoding errors like `RemainingScaleBytesNotEmptyException`.
```

**Comment by LefterisJP** [comment]
Should be fixed by #2345. We will need to test the final binary also with kusama balances!
